Explorer/Assets/DCL/Multiplayer/SDK/Systems/WriteSDKAvatarBaseSystem.cs

```diff
@@ -0,0 +1,94 @@
+using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using Arch.SystemGroups.DefaultSystemGroups;
+using CRDT;
+using CrdtEcsBridge.ECSToCRDTWriter;
+using DCL.Diagnostics;
+using DCL.ECSComponents;
+using DCL.Multiplayer.SDK.Components;
+using DCL.Optimization.Pools;
+using ECS.Abstract;
+using ECS.LifeCycle.Components;
+using ECS.Unity.ColorComponent;
+
+namespace DCL.Multiplayer.SDK.Systems
+{
+    [UpdateInGroup(typeof(PresentationSystemGroup))]
```

Comment:
Should be synced, check other systems

Reply:
Updated them to `SyncedPostRenderingSystemGroup`